src/chainstate/stacks/boot/mod.rs

```diff
@@ -68,6 +68,14 @@ pub fn make_contract_id(addr: &StacksAddress, name: &str) -> QualifiedContractId
     QualifiedContractIdentifier::new(StandardPrincipalData::from(addr.clone()), ContractName::try_from(name.to_string()).unwrap())
 }
 
+pub fn pox_boot_code_constants(mainnet: bool) -> &'static str {
```

Comment:
This can be removed if we create `STACKS_BOOT_CODE_MAINNET` and `STACKS_BOOT_CODE_TESTNET` static arrays with the respective networks' boot codes.

Reply:
I think that'd require something like a lazy_static initializer so that we could still use `format!` to avoid repeating the actual body of the pox contract. I don't think that's preferable here.

Reply:
Addressed in 9d5d1e0d3

Reply:
Thanks!